components/infobox/commons/infobox_person.lua

```diff
@@ -463,4 +469,19 @@ function Person._createAgeCalculationErrorMessage(text)
 	end
 end
 
+function Person:_flipNameOrder(args)
+	if not Logic.readBool(args.nonameflip) and Table.includes(
+		{
```

Comment:
move the table up to the top as a (const) variable

Reply:
Moved this up. (let me know if this was the correct way)